controller/workitem.go

```diff
@@ -107,6 +108,25 @@ func (c *WorkitemController) List(ctx *app.ListWorkitemContext) error {
 		exp = criteria.And(exp, criteria.Equals(criteria.Field(workitem.SystemState), criteria.Literal(string(*ctx.FilterWorkitemstate))))
 		additionalQuery = append(additionalQuery, "filter[workitemstate]="+*ctx.FilterWorkitemstate)
 	}
+	if ctx.FilterCategory != nil {
+		var relationships []*category.CategoryWorkItemTypeRelationship
+		application.Transactional(c.db, func(tx application.Application) error {
+			// Load all workitemtypes related to the specific category
+			relationships, err = tx.Categories().LoadRelationships(ctx, *ctx.FilterCategory)
+			if err != nil {
+				return jsonapi.JSONErrorResponse(ctx, errs.Wrap(err, "Error listing work items"))
```

Comment:
@kwk Okay I will remember to start all error messages with lower case. i will fix this.